cypress/support/index.d.ts

```diff
@@ -490,12 +490,12 @@ declare namespace Cypress {
 
 
     /**
-     * Please help with documentation!
-     * The file where this function is declared appears below, however the parameters in this definition may be wrong. Please adjust as needed.
-     *
+     * It selects the current date from the picker
+     * @fieldName - the date field for which the current date is selected
+     * @modal - boolean, set to true is the date field is in a modal
      * from cypress/support/commands/form.js
      */
-    selectDateViaPicker(fieldName, modal): Chainable<any>
+    selectDateViaPicker(fieldName: string, modal: boolean): Chainable<any>
```

Comment:
Shouldn't `modal` be an optional parameter?
See https://github.com/metasfresh/metasfresh-e2e/blob/8ec5bb3b77ef915b1b092eaa5336ff7af83d4915/cypress/support/index.d.ts#L14-L16

Reply:
done.